packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -154,6 +154,7 @@ class FormatToolbar extends Component {
 					const isActive = isFormatActive || isAddingLink;
 					return {
 						...control,
+						shortcut: isFormatActive ? control.activeShortcut : control.shortcut,
```

Comment:
Should we fallback too `control.shortcut` if `control.activeShortcut` is undefined? sometimes it's the same shortcut to activate/deactive a format right?

Reply:
This code is only specific to the link shortcut, so I haven't worried about a fallback:
https://github.com/WordPress/gutenberg/pull/9903/files#diff-cf43b17f636b0adcea4f5700de06c737R152

Reply:
Oh got it :) It's not obvious at first sight in Github diff :P 

The question is whether we should support this for other formats as block authors will be able to define alternative formats later (with the RichText tree structure work that's being done), it can be addressed later.

Reply:
Hmm, yep, that might be a good idea - the same would need to apply for most of the other properties as well.

Thanks for the review!